packages/cf-component-page/src/PageHeader.js

```diff
@@ -1,22 +1,38 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const PageHeading = createComponent(
+  ({ theme }) => ({
+    ...theme.heading
+  }),
+  'h1'
+);
 
-class PageHeader extends React.Component {
-  render() {
-    return (
-      <header className="cf-page__header">
-        <h1 className="cf-page__title">{this.props.title}</h1>
-        {this.props.subtitle &&
-          <h2 className="cf-page__subtitle">{this.props.subtitle}</h2>}
-      </header>
-    );
-  }
-}
+const PageSubheading = createComponent(
+  ({ theme }) => ({
+    ...theme.subheading
+  }),
+  'h2'
+);
+
+const PageHeader = ({ title, subtitle, className }) => {
+  return (
+    <header className={className}>
```

Comment:
Is the `className` here provided by `fela`? This is the same thing as doing:

```js
createComponent(
  () = > {},
  'header',
  ['title', 'subtitle']
);
```
?